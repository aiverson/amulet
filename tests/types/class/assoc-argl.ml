class foo 'a begin
  type bar 'a
end

instance foo int begin
  type bar = ()
end
