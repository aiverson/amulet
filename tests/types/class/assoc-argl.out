assoc-argl.ml[6:3 ..6:15]: error (E2024)
  │ 
6 │   type bar = ()
  │   ^^^^^^^^^^^^^
  This associated type instance has 0 arguments, but 1 was expected
assoc-argl.ml[6:3 ..6:15]: Use amc explain 2024 for more information
